src/components/layout/Navbar/AuthenticatedNav.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect } from 'react';
+import Materialize from 'materialize-css';
+
+const AuthenticatedNav = () => {
+  useEffect(() => {
```

Comment:
Can we have the states, setStates and use effects in a separate file so that this component file on focuses on rendering the component. The file name can be `AuthenticatedNav.js` while this file becomes `AuthenticatedNav.jsx` as i requested above

Reply:
Noted, Treated Accordingly